joystick_xl/inputs.py

```diff
@@ -236,16 +236,11 @@ def _update(self) -> int:
         new_value = min(max(self._source.value, self._min), self._max)
 
         # account for deadband
-        if new_value < (self._raw_midpoint - self._deadband):
-            new_value -= self._min
-        elif new_value > (self._raw_midpoint + self._deadband):
-            new_value = new_value - self._min - (self._deadband * 2)
-        else:
-            new_value = self._db_range // 2
-
-        # calculate scaled joystick-compatible value and clamp to 0-255
-        self._value = min(new_value * 256 // self._db_range, 255)
```

Comment:
Is it correct that scaling is not supported?
I note that physical and logical maximums are equal, so scaling **_shouldn't_** do anything.   Just making sure...